lib/esm-utils.js

```diff
@@ -1,11 +1,8 @@
-const url = require('url');
 const path = require('path');
 
-const requireOrImport = async file => {
-  file = path.resolve(file);
```

Comment:
Moved `resolve()` to `loadFilesAsync` so the file doesn't have to be a fully-resolved name, but can be a package from node_modules as well.